src/RTT_Format_Reader/RTT_Mesh_Reader.cc

```diff
@@ -116,30 +114,35 @@ void RTT_Mesh_Reader::transform2CGNS(void) {
       }
 
       if (have_quad9) {
-        Insist(have_quad4, "quad or quad4 must appear in the cell_defs "
-                           "block when a quad9 is in a polyhedron");
+        Insist(have_quad4, "quad or quad4 must appear in the cell_defs block "
+                           "when a quad9 is in a polyhedron");
       }
 
+      // Create the unique side element definitions
+      std::vector<int> unique_side_types;
+      std::vector<Element_Definition> elem_defs;
+
       for (unsigned s = 0; s < cell_definition->get_nsides(); ++s) {
         int side_type(cell_definition->get_side_types(s));
 
-        // check to see if this side type has already been added to the
-        // elem_defs list of sides
-        std::vector<int>::iterator sit(
-            std::find(side_types.begin(), side_types.end(), side_type));
-        if (sit == side_types.end()) {
-          // not yet added for this polyhedron, so create an element
-          // of this type
+        // check to see if this side type has already been added to the elem_defs list of sides
```

Comment:
I think that should be automatic with the new formatter. Otherwise I have to count columns
because I don't wrap lines at 80 in my editor automatically.